src/workerd/api/actor-state.c++

```diff
@@ -554,16 +544,12 @@ jsg::Promise<void> DurableObjectStorageOperations::putMultiple(
     // deleting an undefined field is confusing, throwing could break otherwise working code, and
     // a stray undefined here or there is probably closer to what the user desires.
 
-    ActorStorageLimits::checkMaxKeySize(field.name);
-
     kj::Array<byte> buffer = serializeV8Value(js, field.value);
-    ActorStorageLimits::checkMaxValueSize(field.name, buffer);
```

Comment:
Hmm, by doing this means that if we get a giant value we're going to keep it around until the check kicks in later.  I guess this is probably safe, if only because if the value were too huge, the isolate would have run out of memory before we got here.